Libplanet.Tests/Blocks/BlockTest.cs

```diff
@@ -338,19 +338,22 @@ public void EvaluateActionsPerTx()
 
             Transaction<DumbAction>[] blockIdx2Txs =
             {
+                // Note that these timestamps in themselves does not have any meanings but are
+                // only arbitrary.  These purpose to make their evaluation order in a block
```

Comment:
Is there any reason insert two space?

Reply:
AFAIK it's an [old convention][1], and I believe it improves readability on plain text (two spaces eventually will be reduced to one space when it's rendered anyway).

[1]: https://en.wikipedia.org/wiki/Sentence_spacing